phabfive/cli.py

```diff
@@ -130,15 +131,19 @@ def run(cli_args, sub_args):
 
         elif cli_args["<command>"] == "diffusion":
             d = diffusion.Diffusion()
-            if sub_args["repo"] and sub_args["list"]:
-                if sub_args["active"]:
-                    status = ["active"]
-                elif sub_args["inactive"]:
-                    status = ["inactive"]
-                else:
-                    status = REPO_STATUS_CHOICES
-
-                d.print_repositories(status=status, url=sub_args["--url"])
+            if sub_args["repo"]:
+                if sub_args["list"]:
+                    if sub_args["active"]:
+                        status = ["active"]
+                    elif sub_args["inactive"]:
+                        status = ["inactive"]
+                    else:
+                        status = REPO_STATUS_CHOICES
+
+                    d.print_repositories(status=status, url=sub_args["--url"])
+
```

Comment:
Lint: There should not be blank lines before a `elif or else` even in deeply nested logic like this.

Reply:
Fixed